doc/examples/plot_circular_elliptical_hough_transform.py

```diff
@@ -128,10 +128,10 @@
 
 # Estimated parameters for the ellipse
 best = result[-1]
```

Comment:
@jni I updated the commit with your proposed changes. As pointed by @sciunto the code should be both concise and easy to read, which corresponds to your formulation.